src/agoradatatools/great_expectations/gx/expectations/gene_info.json

```diff
@@ -514,25 +516,25 @@
       "kwargs": {
         "column": "biodomains",
         "list_members": [
-          "Apoptosis",
+          "Myelination",
           "Vasculature",
-          "Lipid Metabolism",
-          "Oxidative Stress",
-          "Mitochondrial Metabolism",
-          "APP Metabolism",
-          "Proteostasis",
-          "DNA Repair",
           "Synapse",
           "Immune Response",
-          "Tau Homeostasis",
+          "DNA Repair",
+          "Autophagy",
+          "Endolysosome",
+          "Proteostasis",
+          "Mitochondrial Metabolism",
           "Cell Cycle",
           "Epigenetic",
+          "Lipid Metabolism",
           "Metal Binding and Homeostasis",
-          "Endolysosome",
-          "Structural Stabilization",
-          "Myelination",
           "RNA Spliceosome",
-          "Autophagy"
+          "Tau Homeostasis",
+          "Apoptosis",
+          "Oxidative Stress",
+          "APP Metabolism",
+          "Structural Stabilization"
```

Comment:
Not an issue with your code updates but, any idea why this list keeps getting re-ordered? I noticed it happen on one of Beatriz's PRs that affected this GX. The values are all the same, just in a different order, which makes diffing it less clear. Do you know any way to just auto-alphabetize the list whenever the GX updates or something?

Reply:
Thanks for pointing this out. I don't know about that 🤔 I can look into it but we may just want to watch out for that and try not to commit changes like this one in the future.

Reply:
Yeah after doing some searching I don't think there is anything we can do about it unfortunately. Though it is weird that my machine likes this order and Beatriz's likes the other one... I think we just need to look out for this and try not to commit changes like this one in the future to avoid confusing diffs.